### PR TITLE
Front end [single user data]

### PR DESCRIPTION
**Pull request by Laura-TM**
took changes from other peers on board and adjusted my features

### CODE REVIEW

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -31,19 +31,11 @@ export function Home() {
...
 				<Headers name={"Jim"} cohort={"London-8"} />
 				<div className="card">
 					<h1 className="message" data-qa="message">
 						{message}
 					</h1>
-					{/* <h2>Messages: {statistics[0]}</h2>
-					<h2>Reactions: {statistics[1]}</h2> */}
 				</div>
 				<UserTable
                     userInfo={{ month: "July", posts: statistics[0], reactions: statistics[1], calls: 0 }}
```
Comment (DibaKarimi) [inline client/src/pages/Home.js:41]: Hi Laura,
well done! thank you for your code.
Could you please use end point and real data.

Comment (DibaKarimi) [reply client/src/pages/Home.js:41]: Hi Laura,
well done! Thank you for your code.

File: client/src/pages/Home.js [hunk 2]
```diff
@@ -28,19 +31,15 @@ export function Home() {
 	return (
 		<main role="main">
 			<div>
-				<img
-					className="logo"
-					data-qa="logo"
-					src={logo}
-					alt="Just the React logo"
-				/>
+				<Headers name={"Jim"} cohort={"London-8"} />
```
Comment (DibaKarimi) [inline client/src/pages/Home.js:34]: Could you please use "message" for name?

File: client/src/pages/Home.js (hunk 1 above)
Comment (Laura-TM) [reply client/src/pages/Home.js:41]: Yes, this was already added; will get other live data from other contributors shortly

**Review by DibaKarimi — APPROVED** [review]

**Pull request merged**